CHANGELOG.md

```diff
@@ -27,6 +27,7 @@
 
 * Fixed an issue where changes made to `NavigationViewController.showsReportFeedback`, `NavigationViewController.showsSpeedLimits`, `NavigationViewController.detailedFeedbackEnabled`, `NavigationViewController.floatingButtonsPosition` and `NavigationViewController.floatingButtons` before `NavigationViewController` presentation were not saved. ([#3718](https://github.com/mapbox/mapbox-navigation-ios/pull/3718))
 * Fixed an issue where `SpeechSynthesizing.managesAudioSession` was ignored by `RouteVoiceController`. ([#3572](https://github.com/mapbox/mapbox-navigation-ios/pull/3572))
+* Updated Navigation engine lifecycle, to allow freeing core navigator when there is no active free drive or active guidance sessions. ([#3724](https://github.com/mapbox/mapbox-navigation-ios/pull/3724))
```

Comment:
This description is rather implementation-oriented. For a developer audience, I’d probably say something like this:

> * Location tracking and routing resources are now freed if no `RouteController` or `PassiveLocationManager` instance is actively being used.